esmvalcore/preprocessor/_multimodel.py

```diff
@@ -196,7 +215,7 @@ def _get_overlap(cubes):
     """
     all_times = []
     for cube in cubes:
-        span = _datetime_to_int_days(cube)
+        span = _datetime_to_int_days(cube, overlap=True)
         start, stop = span[0], span[-1]
         all_times.append([start, stop])
     bounds = [range(b[0], b[-1] + 1) for b in all_times]
```

Comment:
I think this can also be simplified:
```Python
    bounds = []
    for cube in cubes:
        span = _datetime_to_int_days(cube)
        start, stop = span[0], span[-1]
        bounds.append(range(start, stop+1))
```
But why even use the range function? Don't we just want the overlapping time points from span?
```Python
    spans = []
    for cube in cubes: 
        spans.append(_datetime_to_int_days(cube))    
    time_pts = reduce(np.intersect1d, spans)
```

Reply:
mate can we please not start trying to fix something that's not broken? We need to understand the very specific problem that @LisaBock is having and to allow for the correct handling of yearly data - the second one works fine so let's just focus on the scope of this PR. I really don't want to start changing this MM right before the release :beer:

Reply:
Sure, that's fair. It's just that I thought it might help to debug, for I'm still struggling to understand these parts of the code.